lib/tasks/commit_fingerprint.rake

```diff
@@ -0,0 +1,24 @@
+namespace :commit_fingerprint do
+  desc "Updates commits with a fingerprint of it's commits_keys"
+  task update: :environment do
+    puts "[commit_fingerprint:update] Updating fingerprints on commits."
+
+    Commit.includes(:commits_keys).all.each do |commit|
```

Comment:
this should use use `find_each` to avoid loading _all_ of the commits into memory at once. Everything else looks good so I will change this myself after merging.

Reply:
Sorry about that, thanks for making the change.